.github/CODEOWNERS

```diff
@@ -0,0 +1,6 @@
+# Each line is a file pattern followed by one or more owners.
+
+# Code owners are automatically added as reviewers on pull requests
+# that contain changes to files they own
+
+/ @BlueSkies7 @daveLambe @dodecaphonix @purplehax @romainrichard
```

Comment:
What about Matt!

Reply:
I don't think he's ready to start yet, he said he'll let me know when he wants me to add him to our GitHub and Jira.